tests/test_recipes.py

```diff
@@ -442,6 +453,20 @@ def test_increment_for_numbers_2(self):
         assert dummy.default_decimal_field == Decimal("23.1")
         assert dummy.default_float_field == 4.23
 
+    def test_increment_for_numbers_with_suffix(self):
+        from model_bakery.recipe import seq  # NoQA
+
+        dummy = baker.make_recipe(
+            "tests.generic.serial_numbers", default_int_field=seq(1, suffix=1)
+        )
+        assert dummy.default_int_field == 3
```

Comment:
If you are a +1 for my previous comment, this should now assert for a `TypeError` too.